src/cursor.c

```diff
@@ -23,6 +23,9 @@ void cursor_move_up(Cursor *cursor, View *view) {
 }
 
 void cursor_move_down(Cursor *cursor, View *view) {
+  if (cursor->y == view->canvas->num_rows - view->y - 1){
```

Comment:
I can't help but feel that checking for `==` instead of `>=`/`<=` will come back to bite us at some point, but it seems fine for now.